packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.story.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import ConnectDialog from './ConnectDialog';
+
+export default {
+  title: 'Teleport/Kubes/ConnectDialog',
+};
+
+export function Connect() {
```

Comment:
```suggestion
export function ConnectDialog() {
```
lets also use arrow function to keep it consistent with other stories

Reply:
The imported component name is `ConnectDialog` as well so I need to change one of them.

Reply:
stories in Nodes are not arrow functions, let me know if it needs changing for consistency.

Reply:
lets leave it as is for now (to make final review of this feature more easier on us), but note that some of our existing code may not be the best or outdated and we seek to improve if we can and have the time to.

Reply:
renaming imported comps is fine, I would've imported it as `Component` even

Reply:
i deleted the other comment, idk why, but lets save refactoring for later so final review of this feature is easier on us. Also note that existing code may not always be best examples or even out dated so we try to improve on them when using them as reference (vs just copy and pasting)